scripts/install-nvidia.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Based on Jonathan Calmels approachhttps://github.com/NVIDIA/nvidia-docker/wiki/Deploy-on-Amazon-EC2
+
+set -xu
+
+# Install NVIDIA drivers 361.42
+sudo apt-get install --no-install-recommends -y gcc make libc-dev
+wget -P /tmp http://us.download.nvidia.com/XFree86/Linux-x86_64/361.42/NVIDIA-Linux-x86_64-361.42.run
+sudo sh /tmp/NVIDIA-Linux-x86_64-361.42.run --silent
+
+# Install nvidia-docker and nvidia-docker-plugin
+wget -P /tmp https://github.com/NVIDIA/nvidia-docker/releases/download/v1.0.0-rc.3/nvidia-docker_1.0.0.rc.3-1_amd64.deb
+sudo dpkg -i /tmp/nvidia-docker*.deb && rm /tmp/nvidia-docker*.deb
+
+# Add login user to docker and nvidia-docker groups
+sudo usermod -aG docker,nvidia-docker ubuntu
+
+# Install our ML 
+git clone https://github.com/albarji/neural-style-docker.git
```

Comment:
Yep!...I was divided about this...This is script is meant to be copied over to the EC2 instance created using "docker-machine scp" and install nvidia drivers, nvidia-docker....and, finally,   added the clone to minimize the number of steps and commands the readers have to type in...we could take it out if you think it´s confusing....(I was trying to beef up the article with docker-machine commands). If we take it out, readers would need to type 

$ docker-machine ssh nvidia-docker
$ git clone https://github.com/albarji/neural-style-docker.git
$ cd neural-style-docker
$ ./scripts/install-nvidia.sh

(I don´t think this is bad either)